tools/build_pak_files.py

```diff
@@ -57,21 +51,44 @@
     # once we know the platform, set our cooked dir
     unreal_project_cooked_dir = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", platform + "NoEditor"))
 
-    if spear.path_exists(unreal_project_content_shared_dir):
-        print(f"[SPEAR | build_pak_files.py] File or directory or symlink exists, removing: {unreal_project_content_shared_dir}")
-        spear.remove_path(unreal_project_content_shared_dir)
+    # We use different strategies for setting scene_ids, depending on if we're creating symlinks or not.
+    # If we're not creating symlinks, then the user must specify args.scene_ids. If we are creating
+    # symlinks, then we get a list of candidate scene_ids from Perforce and optionally filter.
+    if args.skip_create_symlinks:
+        assert args.scene_ids is not None
+        scene_ids = [args.scene_ids]
+    else:
+        assert os.path.exists(args.perforce_content_dir)
```

Comment:
Let's move this up to be logically next to our other validation of input arguments.

```
    assert os.path.exists(args.unreal_engine_dir)
    if not args.skip_create_symlinks:
        assert os.path.exists(args.perforce_content_dir)
```

Reply:
I added assert for `args.scene_ids` in this as well.
```python
    if args.skip_create_symlinks:
        assert args.scene_ids is not None
    else:
        assert os.path.exists(args.perforce_content_dir)
```